<launch>
    <arg name="stdout" default="screen" />
    <arg name="camera_name" default="/zed2i/zed_node/" />
    <arg name="image_topic" default="left_raw/image_raw_color" />
    <arg name="camera_frame_name" default="zed2i_left_camera_optical_frame"/>
    <arg name="lidar_frame_name" default="velodyne"/>
    <arg name="lidar_cloud_name" default="lidar_pattern_0/cumulative_cloud"/>

    <!-- Tranf from lidar to camera-->
    <node pkg="tf" type="static_transform_publisher" name="sensor2_rot_tf" args="0 0 0 -1.57079632679 0 -1.57079632679 rotated_$(arg camera_frame_name) $(arg camera_frame_name) 100" />
    <node pkg="tf" type="static_transform_publisher" name="velo2cam_tf" args="0.478108 -0.595839 -0.660503 -0.0246376 0.00337393 0.0203268 rotated_$(arg camera_frame_name) $(arg lidar_frame_name) 100" />

    <!-- Our node -->
    <node pkg="velo2cam_stereoHough" type="saveImg" name="save_Img" output="screen">
        <remap from="image_left" to="circle_detection_left/left_circles" />
        <remap from="points" to="$(arg lidar_cloud_name)" />
    </node>
</launch>